fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -655,6 +656,11 @@ class HomeFragment : Fragment() {
                     text = it.name,
                     start = DrawableMenuIcon(
                         drawable = it.icon.toDrawable(resources),
+                        tint = if (it.type == SearchEngine.Type.APPLICATION) {
```

Comment:
You will also need to do this in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt#772. Right now, this will only affect the Home screen, but when you navigate to a website and click on the search bar. You will see a mismatch of colors because the search dialog search selector menu isn't updated with the appropriate tint.

As a result, I can see the following after doing a simple search after navigating to a webpage:
<img width="441" alt="Screen Shot 2023-02-23 at 1 26 27 PM" src="https://user-images.githubusercontent.com/1190888/220997416-6dc09d62-68f4-4df9-a191-b7a85a4b1d62.png">